### ISSUE TITLE
ts-guard-decorator

### ISSUE THREAD

**Issue by Olmer1st** [arkon/ng-sidebar#71]
Hi,
after upgrading to latest version of ng-sidebar, application stopped work with error:
```
Failed to load resource: the server responded with a status of 404 (Not Found) :8000/ts-guard-decorator
Error: Fetch error: 404 Not Found
  Instantiating http://localhost:8000/ts-guard-decorator
  Loading http://localhost:8000/lib/ng-sidebar/lib/sidebar.component.js
  Loading http://localhost:8000/lib/ng-sidebar/lib/index.js
```

Any idea?

**Comment by arkon** [comment]
@Olmer1st Yeah, [@siovene found a solution](https://github.com/arkon/ng-sidebar/commit/f2050ba841fead54e5b086a080c1d4475019d261#commitcomment-21864627). I'll be pushing out an update soon to "fix" this.

**Comment by arkon** [comment]
I've published v4.2.1 without ts-guard-decorator.

@Olmer1st Were you also using SystemJS?

**Comment by Olmer1st** [comment]
Great! Thanks. 

Yes. I use systemjs. I don't like webpack
